galaxy_importer/utils/markup.py

```diff
@@ -77,6 +80,19 @@ def get_html(doc_file):
     return None
 
 
+def get_execution_environment(env_file):
```

Comment:
wondering if we should have these in a `utils/yaml.py` module, since yaml isn't markup

Reply:
Makes perfect sense.  Done!